### DISCUSSION TITLE
Consider adding direct download for stable version.

### DISCUSSION THREAD

**Discussion by ghost** [File-New-Project/EarTrumpet#911] [Ideas]
I do not have a windows account, and I refuse to make one just to download this app. 

Dev builds are dev and not stable for a reason, so I'm not interested in those.

Please consider adding a direct download to the eartrumpet.com webpage, or adding releases here on github.

Thank you.

**Comment by riverar** [comment]
> I do not have a windows account, and I refuse to make one just to download this app. 

A Microsoft account is not required to download EarTrumpet from the Microsoft Store. Just close any authentication prompts that may appear and your download will continue.

**Reply by PanoptesDreams** [reply]
Many people, self included, do not want to go into the Microsoft store at all. We know what apps we want, we have installers/scripts for them and we don't want to have to touch the MS store.

**Reply by riverar** [reply]
@PanoptesDreams Then you can add to your script the ability to download and extract the nuget package from Chocolatey https://community.chocolatey.org/packages/eartrumpet/. Right?

**Reply by PanoptesDreams** [reply]
What I've been doing is extracting the release.zip that comes from chocolatey package, putting it where it *belongs* and doing what your chocolatey script does but without the need to rely on chocolatey software.
Plus you are removing control from the user, I don't want a start menu shortcut..

Why can't you just put that 'release.zip' into the release section here on GitHub?

**Reply by riverar** [reply]
Several reasons. We'll lose the ability to deliver updates and service the application (think security, crashing on boot issues, etc.). Posting binaries on GitHub will cause confusion when the app doesn't automatically update and we don't want to build out an entire servicing pipeline when better alternatives exist.

**Reply by PanoptesDreams** [reply]
The only people here are developers, we know what we are doing (in theory). Maybe we don't need the app to auto update, If I want updates I 'could' pull them straight from GitHub. And I don't see this causing security issues or hard crashes, unlike Apple products; It just works. No seriously Ear Trumpet has been magical, never had a single issue.

For everyone else they can just stick to mainline channels. (MS Store, and Choco)

Plus this doubles as a version archive, some people may want particular versions for one reason or another, new version buggy and MS store got you stuck on latest build, no problem, remove it and download an old build (Or in my case pull it from my software cache)

**Reply by riverar** [reply]
I understand it's a pain for scripting, so how about we meet half-way? I can look into uploading the master artifacts to something like `install.eartrumpet.app/master/eartrumpet_x.x.x.x.zip`. Would that be usable?

**Reply by PanoptesDreams** [reply]
That would be insanely helpful.
And adding this to the docs so other developers know they can pull from a 'clean'/safe source.

Thank you

**Reply by riverar** [reply]
OK will get this done hopefully tonight/tomorrow and notify you. 🤠
